### PR TITLE
Typos from LO of June

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6946,6 +6948,7 @@ listensers->listeners
 listernes->listeners
 listner->listener
 listners->listeners
+liszt->list
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: This is a name of a (somewhat?) famous composer, do you think we should disable it by default? Although this is "codespell" so I am a bit torn on this one, I imagine the false alarms will be much rarer than the true positives for this one

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12494,6 +12499,7 @@ wirh->with
 wirte->write
 wirtes->writes
 wirting->writing
+wit hthe->with the
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: I think currently we cannot find errors with spaces in them, can you check?

If we can't, are you up for modifying `test_dictionary_formatting` to ensure no whitespace characters are in `err`?

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: You are right. No way to match word with space, right now.
I remove this. I'm going to add a few commits, so I can empty my LO queue.

Thanks a lot,
Andrea

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: you can always just do 
`liszt->list, liszt,`

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Thanks a lot @luzpaz . I'm going to upload new version

**Pull request merged**